superdesk/io/feeding_services/twitter.py

```diff
@@ -103,6 +100,11 @@ def _update(self, provider, update, test=False):
                 elif exc.message[0].get('code') == 32:
                     # invalid credentials
                     raise IngestTwitterError.TwitterLoginError(exc, provider)
+                elif exc.message[0].get('code') == 88:
+                    # rate limit exceeded
+                    raise IngestTwitterError.TwitterRateLimitError(exc, provider)
+                else:
+                    raise IngestApiError.apiGeneralError(exc, provider)
```

Comment:
Should we add TwitterAPIGeneralError instead of using apiGeneralError from IngestApiError?

Reply:
Since this API error is twitter related I would add `TwitterAPIGeneralError `